x-pack/test/functional/services/transform/wizard.ts

```diff
@@ -139,12 +146,14 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
           `EuiDataGrid rows should be '${expectedNumberOfRows}' (got '${rowsData.length}')`
         );
 
-        rowsData.map((r, i) =>
-          expect(r).to.length(
-            columns,
-            `EuiDataGrid row #${i + 1} column count should be '${columns}' (got '${r.length}')`
-          )
-        );
+        // cell virtualization means the last column is cutoff in the functional tests
+        // https://github.com/elastic/eui/issues/4470
+        // rowsData.map((r, i) =>
+        //   expect(r).to.length(
+        //     columns,
+        //     `EuiDataGrid row #${i + 1} column count should be '${columns}' (got '${r.length}')`
+        //   )
+        // );
```

Comment:
Do you think this could be flaky in the future or would it be safe to count and assert the visible elements?

Reply:
It should be stable, but based on window size & page layout; if the test config modifies the browser size, or if the DOM changes in any meaningful way, it'll require an update to the test